### DISCUSSION TITLE
docker docs [Nov 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4365] [Discord Threads]
> 🙋‍♂️ **McPizza**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### you can see the example here   
#### https://docs.nocodb.com/getting-started/installation#docker  
![image0](https://cdn.discordapp.com/attachments/1034961918819049472/1035028654528405554/unknown.png)  
#  
> 🙋‍♂️ **wkw**  
#### > didn't work   
####    
#### can u elaborate more? like any errors u've seen? are you using localhost instead of host.docker.internal?  
#  
> 🙋‍♂️ **McPizza**  
#### Thats the string I tried, but it was timing out   
####    
#### Didnt copy the error, but its was something about the connection timing out   
####    
#### without the `&t1` it didnt work   
#### The address was for an external postgres server, using it's url   
####    
#### Now I removed the `&t1`from the url, and its working fine.   
####    
#### Might just be on first boot/install  
#  
> 🙋‍♂️ **wkw**  
#### &t flag has been removed in latest version  
#  
> 🙋‍♂️ **wkw**  
#### is your Postgres running on your local machine ?  
#  
> 🙋‍♂️ **wkw**  
#### Also docker? Or on cloud?  
#  
> 🙋‍♂️ **McPizza**  
#### also docker  
#  
> 🙋‍♂️ **McPizza**  
#### trying to replicate now with fresh instance  
#  
> 🙋‍♂️ **wkw**  
#### Ok  
#  
> 🙋‍♂️ **McPizza**  
#### Got it  
#  
> 🙋‍♂️ **McPizza**  
#### ````   
####    
#### 2022-10-27T12:19:22.404673166Z Visit -> http://localhost:8080/dashboard   
#### 2022-10-27T12:19:22.404617591Z App started successfully.   
#### 2022-10-27T12:19:22.623612311Z ╔══════════════════════════════════════════════════════════════════════════════╗║ ║║ EXCEPTION OCCURED!! in PGClient @ createDatabaseIfNotExists ║║ - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - ║║ - - - - - - - - - - - - - - - - - - - - - - ║║ MESSAGE: ║║ CREATE SCHEMA IF NOT EXISTS "public" AUTHORIZATION "spore2985" - ║║ permission denied for database noco2 ║║ ║║ - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - ║║ - - - - - - - - - - - - - - - - - - - - - - ║║ CODE: ║║ 42501 ║║ ║║ - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - ║║ - - - - - - - - - - - - - - - - - - - - - - ║║ STACK: ║║ error: CREATE SCHEMA IF NOT EXISTS "public" AUTHORIZATION "spore2985" ║║ - permission denied for database noco2 ║║ at Parser.parseErrorMessage ║║ (/usr/src/app/node_modules/pg-protocol/dist/parser.js:287:98) ║║ at Parser.handlePacket ║║ (/usr/src/app/node_modules/pg-protocol/dist/parser.js:126:29) ║║ at Parser.parse ║║ (/usr/src/app/node_modules/pg-protocol/dist/parser.js:39:38) ║║ at Socket.<anonymous> ║║ (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42) ║║ at Socket.emit (events.js:314:20) ║║ at Socket.EventEmitter.emit (domain.js:483:12) ║║ at addChunk (_stream_readable.js:297:12) ║║ at readableAddChunk (_stream_readable.js:272:9) ║║ at Socket.Readable.push (_stream_readable.js:213:10) ║║ at TCP.onStreamRead (internal/stream_base_commons.js:188:23) ║║ ║║ - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - ║║ - - - - - - - - - - - - - - - - - - - - - - ║║ ║║ ║╚══════════════════════════════════════════════════════════════════════════════╝   
####    
#### ````  
#  
> 🙋‍♂️ **wkw**  
#### Is noco2 created by u before starting nocodb?  
#  
> 🙋‍♂️ **McPizza**  
#### ah wait, forgot to add permissions  
#  
> 🙋‍♂️ **wkw**  
#### Seems permission issue  
#  
> 🙋‍♂️ **McPizza**  
#### hmmm... now its working  
#  
> 🙋‍♂️ **McPizza**  
#### yesterday was separate issue saying connection timed out  
#  
> 🙋‍♂️ **wkw**  
#### Cool  
#  
> 🙋‍♂️ **McPizza**  
#### so maybe I had a stale docker image¿  
#  
> 🙋‍♂️ **Flame**  
#### NocoDB throws connection timeout error for any failure in the Database Connection so it was likely that your string was incorrect.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1034961918819049472)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
